addons/point_of_sale/models/pos_order.py

```diff
@@ -854,6 +854,9 @@ def _prepare_aml_values_list_per_nature(self):
                     'amount_currency': payment_id.amount,
                     'balance': self.session_id._amount_converter(payment_id.amount, self.date_order, False),
                 })
+        # Payment related entries should not be excluded from follow-ups
+        for aml_vals in aml_vals_list_per_nature['payment_terms']:
+            aml_vals['blocked'] = False
```

Comment:
Why not putting it in the values directly when appending to `aml_vals_list_per_nature['payment_terms']`?

Reply:
The append right above happens in an `else` branch (inside a `for` loop), so the "unblock" logic would have to be split. I think it is cleaner to just "unblock" all 'payment_terms' lines after the `for` loop. Then the logic is in one place and we do not have to care about what happens in the `for` loop exactly.

Reply:
All the values in `aml_vals_list_per_nature['payment_terms']` are coming from that `else` branch, they are only modified in the other branch

Reply:
Good point thanks